plugins/bazaar-backend/migrations/20221121120212_add_title.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line func-names
+exports.up = async function (knex) {
+  await knex.schema
+    .alterTable('metadata', table => {
+      table.string('title').comment('The title of the entity');
+    })
+    .then(() => knex('metadata').update('title', knex.ref('name')));
```

Comment:
could have ended the previous statement, and put the update as a second awaited statement, but this works too!